### PR TITLE
dobi: init at 0.8

### PR DESCRIPTION
**Pull request by vdemeester**
###### Motivation for this change

[dobi](https://github.com/dnephin/dobi) is a build automation tool for Docker applications that I tend to use on several projects.

I have a question though : Is there a way to specify `-ldflags` with `buildGoPackage` ? (didn't find it but I might have miss something).

###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] OS X
   - [ ] Linux
- [x] <del>Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`</del>
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

🐸

Signed-off-by: Vincent Demeester <vincent@sbr.pm>

### CODE REVIEW

**Comment by kamilchm** [comment]
You could try to pass `-ldflags` to [`buildFlags`](https://github.com/NixOS/nixpkgs/blob/98a9d815e05dd56aba0f9040d96092335cd90444/pkgs/applications/networking/syncthing012/default.nix#L8)

**Comment by vdemeester** [comment]
ooh I missed that one :sweat_smile: Thanks @kamilchm, I'll do that \o/

**Review by fpletz — CHANGES_REQUESTED** [review]
This package generates a `$out/bin/script` binary. What's that used for? It's a very generic name and seems to fail to load some config to generate documentation?

If it's not useful for using dobi, please simply remove it for instance in the `postInstall` hook.

**Comment by vdemeester** [comment]
@fpletz updated by adding `docs` as *excluded package* (as it's not needed here)

File: pkgs/development/tools/dobi/default.nix [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{ stdenv, lib, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  name = "dobi-${version}";
+  version = "v0.8";
```
Comment (joachifm) [inline pkgs/development/tools/dobi/default.nix]: Version strings must begin with a digit. In this case, the package will end up with the name "dobi-v0.8" and no version (see `builtins.parseDrvName`).

Comment (vdemeester) [inline pkgs/development/tools/dobi/default.nix]: ohh 😅

**Pull request closed without merging**